app/src/main/res/layout/actor_card.xml

```diff
@@ -23,5 +24,6 @@
         android:textColor="@color/white_gray"
         android:textSize="@dimen/textsize_12"
         android:textStyle="bold"
+        tools:ignore="SpUsage"
```

Comment:
Why you don't use SP?

Reply:
We have already discussed it with Nikita, I won't use sp in this project to prevent design shift